src/main/java/io/github/jpmorganchase/fusion/api/operations/APIUploadOperations.java

```diff
@@ -1,27 +1,27 @@
-package io.github.jpmorganchase.fusion.api;
+package io.github.jpmorganchase.fusion.api.operations;
 
 import io.github.jpmorganchase.fusion.api.exception.APICallException;
 import java.io.InputStream;
 
-public interface APIUploader {
+public interface APIUploadOperations {
 
-    void callAPIFileUpload(
+    default void callAPIFileUpload(
```

Comment:
Why do we use this pattern?

Reply:
You still want to have a conversation about this?

Reply:
Can be merged now, let's just talk about that later